cli/src/main.rs

```diff
@@ -88,6 +88,13 @@ enum Command {
         #[clap(long, default_value = "false")]
         dry_run: bool,
     },
+    /// marks backend(s) as lost.
+    Abandon {
```

Comment:
Can we name this something like `MarkLost` or `MarkBackendLost`, for clarity?

Reply:
yessir.